Quadriga/src/main/webapp/WEB-INF/views/public/nav/nav-networkrightplane.jsp

```diff
@@ -1,6 +1,8 @@
 <%@ page language="java" contentType="text/html;"%>
 <%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c"%>
+<%@ page import = "edu.asu.spring.quadriga.domain.impl.workspace.TextFile" %>
 
+<script src="${pageContext.servletContext.contextPath}/resources/js/cytoscape/publicNetwork.js" ></script>
```

Comment:
Yes, it is already included in the main jsp. Removed the script from nav-networkrightplane.jsp.